docs/CONTRIBUTING.md

```diff
@@ -1,21 +1,147 @@
-# Contributing
+# Contributing to the web-features project
 
-Thanks for your interest in contributing to this project!
-Before you contribute, consider the following:
+Thanks for your interest in contributing to this project! Before you contribute, consider the following:
```

Comment:
For what it's worth, I like (but will not insist on) semantic line breaks. https://sembr.org/ 😄